facebook/delphiFacebook/R/count.R

```diff
@@ -59,6 +59,7 @@ compute_count_response <- function(response, weight, sample_size)
 #' @param df Data frame
 #' @return Updated data frame.
 #' @importFrom dplyr mutate
+#' @export
```

Comment:
Don't need this if we move the table definitions inside the library as recommended above